index.html

```diff
@@ -881,14 +886,29 @@ <h3>
       </li>
       <li>
         <p>
-          Otherwise, instruct the [= ICE agent =] to remove the candidate pair indicated by |candidatePair|.
+          Otherwise (if |accepted| is <code>true</code>), run the following steps:
         </p>
+        <ol>
+          <li>
+            <p>
+              [= list/Remove =] all items that [= candidate pair match | match =] |candidatePair| from |transport|.{{RTCIceTransport/[[CandidatePairs]]}}.
+            </p>
+          </li>
+          <li>
+            <p>
+              Instruct the [= ICE agent =] to remove the candidate pair indicated by |candidatePair|.
+            </p>
+          </li>
+        </ol>
       </li>
     </ol>
     <p>
-      The {{RTCIceTransport}} object is extended by adding the following internal slot:
+      The {{RTCIceTransport}} object is extended by adding the following internal slots:
     </p>
     <ul>
+      <li>
+        <dfn data-dfn-for="RTCIceTransport">[[\CandidatePairs]]</dfn> initialized to an empty list.
```

Comment:
Normally I'd say use an [ordered set](https://infra.spec.whatwg.org/#ordered-set) instead, but I guess a list is fine since implementations can't rely on object match anyway since pairs are dictionaries with a match algorithm.

This means it's up to algorithms to keep out duplicates.